poppler-sys/src/lib.rs

```diff
@@ -1,28 +1,25 @@
 #![allow(
-    dead_code,
     non_upper_case_globals,
     non_camel_case_types,
     non_snake_case,
     improper_ctypes
 )]
-
-extern crate cairo;
-extern crate cairo_sys;
-extern crate gio_sys;
-extern crate glib;
-extern crate glib_sys;
-extern crate gobject_sys;
-extern crate gtypes;
-
+#[warn(
+    non_upper_case_globals,
+    non_camel_case_types,
+    non_snake_case,
+    improper_ctypes
+)]
 mod dep_types {
     pub use cairo_sys::{cairo_region_t, cairo_surface_t, cairo_t};
     pub use glib_sys::{
-        gboolean, gpointer, GArray, GDate, GError, GList, GQuark, GString, GTime, GType, GTree, GBytes
+        gboolean, gpointer, GArray, GBytes, GDate, GError, GList, GQuark, GString, GTime, GTree,
+        GType,
     };
     pub use gobject_sys::{GObject, GObjectClass};
     pub use gtypes::{gchar, gdouble, gint, gsize, guint, gushort};
-    pub type guint64 = u64;
```

Comment:
this definition was no longer necessary because that type `guint64` is now whitelisted on `WHITELIST_POPPLER_MOVIE`.